bids-validator-web/components/Issues.jsx

```diff
@@ -1,91 +1,92 @@
   constructor(props) {
-    super(props)
-    this._reset = this.props.reset.bind(this)
+    super(props);
+    this._reset = this.props.reset.bind(this);
   }
   // life cycle events --------------------------------------------------
 
   render() {
     // short references
-    let errors = this.props.errors
-    let warnings = this.props.warnings
-    let dirName = this.props.dirName
+    let errors = this.props.errors;
+    let warnings = this.props.warnings;
+    let dirName = this.props.dirName;
 
     // counts
-    let totalErrors = 0
-    let totalWarnings = 0
-    let warningCount, errorCount
-    if (errors !== 'Invalid') {
-      totalErrors = errors.length
-      totalWarnings = warnings.length
-      warningCount = totalWarnings + ' ' + pluralize('Warning', totalWarnings)
-      errorCount = totalErrors + ' ' + pluralize('Error', totalErrors)
+    let totalErrors = 0;
+    let totalWarnings = 0;
+    let warningCount, errorCount;
+    if (errors !== "Invalid") {
+      totalErrors = errors.length;
+      totalWarnings = warnings.length;
+      warningCount = totalWarnings + " " + pluralize("Warning", totalWarnings);
+      errorCount = totalErrors + " " + pluralize("Error", totalErrors);
     }
     // messages
     let specLink = (
       <h5>
-        Click to view details on{' '}
+        Click to view details on{" "}
         <a href="http://bids.neuroimaging.io" target="_blank">
           BIDS specification
         </a>
       </h5>
-    )
+    );
     let notBIDSMessage = (
       <h4>
         This directory failed an initial Quick Test. This means the basic names
         and structure of the files and directories do not comply with BIDS
-        specification. <span onClick={this._reset}>Select a new folder</span>{' '}
+        specification. <span onClick={this._reset}>Select a new folder</span>{" "}
         and try again.
       </h4>
-    )
-    let warningsMessage = <h4>We found {warningCount} in your dataset.</h4>
-    let errorMessage = <h4>Your dataset is not a valid BIDS dataset.</h4>
-    let noErrorMessage = <h4>This is a valid BIDS dataset!</h4>
+    );
+    let warningsMessage = <h4>We found {warningCount} in your dataset.</h4>;
+    let errorMessage = <h4>Your dataset is not a valid BIDS dataset.</h4>;
+    let noErrorMessage = <h4>This is a valid BIDS dataset!</h4>;
     let neurostarsLink = (
       <h5>
-        If you have any questions please post on{' '}
+        If you have any questions please post on{" "}
         <a href="https://neurostars.org/tags/bids" target="_blank">
           Neurostars
         </a>
       </h5>
-    )
+    );
     let sourcecode = (
       <h5>
-        The source code for the validator can be found{' '}
+        The source code for the validator can be found{" "}
         <a
-          href="https://github.com/INCF/bids-validator/tree/gh-pages"
-          target="_blank">
+          href="https://github.com/bids-standard/bids-validator/tree/gh-pages"
```

Comment:
looks good to me ... HOWEVER, why is this pointing to the `gh-pages` branch and not `master`?

Reply:
I also wondered about that, but did not dare to touch it. For me it makes more sense to point to the top level of the <https://github.com/bids-standard/bids-validator> repository.

Reply:
Now that you are at it: I also propose to change the link at https://github.com/bids-standard/bids-validator/blob/d93f054ff8567243982b438991c83911c72a54c3/bids-validator-web/components/Issues.jsx#L38 to https://bids-specification.readthedocs.io where the current spec is.

Reply:
That link is under "Click to view details on BIDS specification" here.

<img width="472" alt="Screen Shot 2019-06-04 at 20 19 46" src="https://user-images.githubusercontent.com/899043/58903625-27df2600-8706-11e9-96bc-04ccdc5f491d.png">

Reply:
yes, sounds reasonable!

```suggestion
          href="https://github.com/bids-standard/bids-validator"
```